android/src/main/java/com/rncamerakit/CKCamera.kt

```diff
@@ -180,13 +185,30 @@ class CKCamera(context: ThemedReactContext) : FrameLayout(context), LifecycleObs
             orientationListener!!.enable()
 
             val scaleDetector =  ScaleGestureDetector(context, object: ScaleGestureDetector.SimpleOnScaleGestureListener() {
-                override fun onScale(detector: ScaleGestureDetector): Boolean {
+                override fun onScaleBegin(detector: ScaleGestureDetector?): Boolean {
+                    val cameraZoom = camera?.cameraInfo?.zoomState?.value?.zoomRatio ?: return false
+                    detector ?: return false
+                    zoomStartedAt = cameraZoom
+                    pinchGestureStartedAt = detector.currentSpan
+                    return true
+                }
+                override fun onScale(detector: ScaleGestureDetector?): Boolean {
```

Comment:
I think this may have reverted a change made in https://github.com/teslamotors/react-native-camera-kit/pull/551/files?w=1

Reply:
Thanks @elliottkember - This was re-fixed and released in v14.0.0-beta13.